python/tk_multi_workfiles/file_save_form.py

```diff
@@ -68,13 +68,20 @@ def __init__(self, parent=None):
             preview_colour = font_colour.darker(140)
         self._preview_colour = (preview_colour.red(), preview_colour.green(), preview_colour.blue())
 
+        self._initializing = False
         try:
             # doing this inside a try-except to ensure any exceptions raised don't 
             # break the UI and crash the dcc horribly!
+            self._initializing = True
```

Comment:
Is this needed? Looks like this would be equivalent:

```
self._initializing = True
try:
   do_stuff()
except:
   self._initializing = False
```

or maybe even

```
self._initializing = True
try:
   self._do_init()
except:
   app.log_exception("Unhandled exception during File Save Form construction!")
finally:
   self._initializing = False

self._start_preview_update()
```